TimberAPI/SpecificationSystem/Fixes/CustomSpecifications/README.md

```diff
@@ -0,0 +1,9 @@
+# TimberAPI custom specifications
+Provide specifications to work around limitations for modders. These should be removed whenever the base game supports it!
```

Comment:
What limitations? 
Should be removed when the game supports what?
These are unclear from current text